src/Computation.hs

```diff
@@ -65,13 +66,43 @@ computeSurjection g numPoints (lowerx, lowery) (topx, topy) =
                   else let (xs_inner, g') = randomsIn g (numPoints - 2) (r2f lowerx, r2f topx)
                            xs = lowerx : xs_inner ++ [topx] -- Include endpts so function covers domain
                            xs_increasing = sort xs
-
                            (ys_inner, g'') = randomsIn g' (numPoints - 2) (r2f lowery, r2f topy) 
                            ys = lowery : ys_inner ++ [topy] --clude endpts so function is onto
                            ys_perm = shuffle' ys (length ys) g'' in -- Random permutation. TODO return g3?
-
                            (zip xs_increasing ys_perm, g'') -- len xs == len ys
 
+
+-- Given a generator, number of points, and lower left and top right of bbox, return points for a bijection.
+-- Points generated lie in the bbox given, whether in math space or screen space
+-- TODO pass randomness around in Runtime
+computeBijection :: Autofloat a => StdGen -> Integer -> Pt2 a -> Pt2 a -> ([Pt2 a], StdGen)
+computeBijection g numPoints (lowerx, lowery) (topx, topy) = 
+                  if numPoints < 2 then error "Bijection needs to have >= 2 points" 
+                  else let (xs_inner, g') = randomsIn g (numPoints - 2) (r2f lowerx, r2f topx)
+                           xs = lowerx : xs_inner ++ [topx] -- Include endpts so function covers domain
+                           xs_increasing = nub (sort xs)
+                           (ys_inner, g'') = randomsIn g' (numPoints - 2) (r2f lowery, r2f topy) 
+                           ys = lowery : ys_inner ++ [topy] --clude endpts so function is onto
+                           ys_increasing = nub (sort ys) in -- Random permutation. TODO return g3?
```

Comment:
Try to generate a more thorough sampling of bijections (e.g. monotonically decreasing as well)